.eslintrc.yml

```diff
@@ -1,47 +1,10 @@
 extends:
-  - airbnb
-  - prettier
-  - prettier/react
+  - '@oacore/eslint-config-react'
```

Comment:
I agree with this line but I do not really support the config itself. I will add a PR there tomorrow.